README.md

```diff
@@ -103,14 +91,20 @@ Available packages are listed in the [packages directory](https://github.com/chr
 
 Chromebrew will wipe its `BREW_DIR` (`/usr/local/tmp/crew` by default) after installation unless you pass `-k` or `--keep` when running `crew install`.
 
-```bash
+```text
```

Comment:
The `bash` passed with the previous rules?

Reply:
Yes but the `<` and `[` symbol will break syntax highlighting